rai/node/bootstrap.hpp

```diff
@@ -79,6 +79,7 @@ class bootstrap_attempt : public std::enable_shared_from_this<bootstrap_attempt>
 	void requeue_pull (rai::pull_info const &);
 	bool still_pulling ();
 	std::deque<std::weak_ptr<rai::bootstrap_client>> clients;
+	std::weak_ptr<rai::bootstrap_client> connection_frontier_request;
```

Comment:
Could this be renamed to indicate it's a lock? Would make the usage code a bit clearer.